linux/power_operation_script/set_power_script_default.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Set and check default soft power operation
+# # Parameters:
+#   power_ops: power operation from command vmware-toolbox-cmd
+# # Return:
+#   script_file_path: path of customized script file
+
+- name: "Get default power script file path"
+  ansible.builtin.command: "{{ vmware_toolbox_cmd_path }} script {{ power_ops }} default"
+  register: power_script_path_result
+  delegate_to: "{{ vm_guest_ip }}"
+
+- name: "Check script file exists"
+  ansible.builtin.stat:
+    path: "{{ power_script_path_result.stdout.strip() }}"
+  delegate_to: "{{ vm_guest_ip }}"
+  register: script_file_exist_result
+  failed_when: not script_file_exist_result.stat.exists
```

Comment:
`failed_when:  script_file_exist_result.stat.exists is undefined or not script_file_exist_result.stat.exists`